java/integTest/src/test/java/glide/CommandTests.java

```diff
@@ -0,0 +1,38 @@
+package glide;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import java.util.concurrent.TimeUnit;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CommandTests {
+
+    private static RedisClient regularClient = null;
+
+    @BeforeAll
+    @SneakyThrows
+    public static void init() {
+        regularClient =
+                RedisClient.CreateClient(
+                                RedisClientConfiguration.builder()
+                                        .address(NodeAddress.builder().port(TestConfiguration.STANDALONE_PORT).build())
+                                        .build())
+                        .get(10, TimeUnit.SECONDS);
+    }
+
+    @AfterAll
+    @SneakyThrows
+    public static void deinit() {
```

Comment:
`teardown`?

Reply:
The name does not matter, junit discovers function by the annotation.

Reply:
still, better to use standard names.